doc/idlist.md

```diff
@@ -85,6 +85,7 @@
 | illumos	| 25	| smmsp
 | extra		| 27	| postfix
 | extra		| 28	| postdrop
+| extra     | 35    | sdns
```

Comment:
It looks like you misinterpreted the input from @citrus-it. Yes we need this in groups since you allocate the `sdns` gid. But you'll also need to leave your initial entry in users, as you allocate a uid as well